docs/concepts/resource_flavor.md

```diff
@@ -0,0 +1,87 @@
+# ResourceFlavor object
```

Comment:
```suggestion
# Resource Flavor
```

for consistency with the other pages. Or maybe they should be titled `ClusterQueue` and `LocalQueue` instead? like  https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/